CHANGELOG.md

```diff
@@ -20,6 +20,10 @@
 * **[Fix]** Fix duplicate in-app update dialog when restarting (or switching) activity quickly after clicking download. Also fixes a crash when choosing "Ask me in a day" in the duplicate dialog.
 * **[Fix]** Fix a crash that could occur when downloading the update with a customized dialog and then calling `Distribute.notifyUserConfirmation(UpdateAction.POSTPONE)` right after calling `Distribute.notifyUserConfirmation(UpdateAction.UPDATE)`.
 
+### App Center Push
+
+* **[Fix]** Update firebase dependency to avoid conflict in applications using latest support libraries (v28) or the latest Firebase messaging version (v18).
```

Comment:
Capital letter *F*
```suggestion
* **[Fix]** Update Firebase dependency to avoid conflict in applications using latest support libraries (v28) or the latest Firebase messaging version (v18).
```